### DISCUSSION TITLE
How to point forge to existing model folders?

### DISCUSSION THREAD

**Discussion by Traslogan** [lllyasviel/stable-diffusion-webui-forge#304] [Q&A]
I've got various WebUI tools, and usually I point them all to the same folder for my SD1.5 / SDXL models to save on space when there's lots of them. I can't seem to find an option to point it to a different directory on Forge. Is this not possible in the current version?

**Comment by catboxanon** [comment]
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/206

**Comment by aleph23** [comment]
I edited my config.json.

The added cmdline param didn't catch everything I needed moved.
